bluemira/equilibria/coils/_grouping.py

```diff
@@ -532,6 +537,7 @@ def get_max_current(self, max_current: float = np.inf) -> np.ndarray:
         return np.where(
             np.isnan(self.j_max) | ~self._flag_sizefix,  # or not
             max_current,
+            # todo: these are np arrays, not floats
             get_max_current(self.dx, self.dz, self.j_max),
```

Comment:
Whats todo?

Reply:
Was a note to correct the types for the func. params.
I'll update